src/services/news/index.js

```diff
@@ -1,3 +1,5 @@
+const express = require('@feathersjs/express');
```

Comment:
Codacy found an issue: [File has too many classes (2). Maximum allowed is 1.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6067861)